### PR TITLE
ADD OpenGridNode repo

### PR DESCRIPTION
**Pull request by IonesioJunior**
## Description
ADD reference to a different repository who stores multiple network instance addresses.

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: src/syft/grid/duet/__init__.py [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
-# syft relative
 from .webrtc_duet import Duet as WebRTCDuet  # noqa: F811
 
 nest_asyncio.apply()
 
-WebRTC_HOST = (
-    "http://ec2-18-191-23-46.us-east-2.compute.amazonaws.com:5000"  # noqa: F811
+
+ADDR_REPOSITORY = (
+    "https://raw.githubusercontent.com/OpenMined/OpenGridNodes/master/network_address"
```
Comment (madhavajay) [inline src/syft/grid/duet/__init__.py:21]: awesome idea

**Review by madhavajay — APPROVED** [review]
LGTM! Great idea!

File: src/syft/grid/duet/__init__.py [hunk 2]
```diff
@@ -135,6 +147,8 @@ def launch_duet(
     print("♫♫♫ >")
```
Comment (iamtrask) [inline src/syft/grid/duet/__init__.py:147]: can we move the print statement for the network_url to be after it finds it?

Comment (iamtrask) [reply src/syft/grid/duet/__init__.py:147]: basically i want both clients to be able to know what network url they're using but this prints it first.

**Review by iamtrask — CHANGES_REQUESTED** [review]
small change

File: src/syft/grid/duet/__init__.py [hunk 3]
```diff
@@ -213,6 +227,8 @@ def join_duet(
     print("♫♫♫ > " + str(network_url))
     print("♫♫♫ >")
```
Comment (iamtrask) [inline src/syft/grid/duet/__init__.py:228]: same here - can we print the network_url after we call get_available_network()

**Review by tudorcebere — APPROVED** [review]

File: src/syft/grid/duet/__init__.py (hunk 2 above)
Comment (madhavajay) [reply src/syft/grid/duet/__init__.py:150]: Looks good

**Comment by madhavajay** [comment]
Looks ready to go!

**Pull request merged**